fontbe/src/post.rs

```diff
@@ -0,0 +1,35 @@
+//! Generates a [post](https://learn.microsoft.com/en-us/typography/opentype/spec/post) table.
+
+use fontdrasil::orchestration::Work;
+use read_fonts::types::{FWord, Fixed};
+use write_fonts::tables::post::Post;
+
+use crate::{
+    error::Error,
+    orchestration::{BeWork, Context},
+};
+
+struct PostWork {}
+
+pub fn create_post_work() -> Box<BeWork> {
+    Box::new(PostWork {})
+}
+
+impl Work<Context, Error> for PostWork {
+    /// Generate [post](https://learn.microsoft.com/en-us/typography/opentype/spec/post)
+    fn exec(&self, context: &Context) -> Result<(), Error> {
+        // TODO a more serious post
+        let post = Post::new(
+            Fixed::from_i32(0),
+            FWord::new(0),
+            FWord::new(16),
```

Comment:
what is 16 here? I think for a viable MVP post format 3 you just need to set format to 3.0 and all the rest of the fields to 0

Reply:
looks like the third new() parameter is the post.underlineThickness

https://github.com/googlefonts/fontations/blob/5fc746797c643c04b1dee2091ceb8f9642c84e37/write-fonts/generated/generated_post.rs#L79

I think 16 is too arbitrary. If you don't want to set to 0, then you can do what ufo2ft does (uses a ratio of the upem):

https://github.com/googlefonts/ufo2ft/blob/5b22f201b4650ab9ef1169357702a732e3127a5e/Lib/ufo2ft/fontInfoData.py#L313-L322

Reply:
I switched to your earlier suggestion of defaults and filed https://github.com/googlefonts/fontmake-rs/issues/146 so I don't forget where to get better underline values from.